packages/scaffold/src/partials/w3m-social-login-widget/index.ts

```diff
@@ -162,32 +160,21 @@ export class W3mSocialLoginWidget extends LitElement {
       RouterController.push('ConnectingSocial')
     }
     const authConnector = ConnectorController.getAuthConnector()
-    if (CoreHelperUtil.isMobile()) {
-      this.mobileWindow = CoreHelperUtil.returnOpenHref(
-        '',
-        'popupWindow',
-        'width=600,height=800,scrollbars=yes'
-      )
-    }
+    this.popupWindow = CoreHelperUtil.returnOpenHref(
+      '',
+      'popupWindow',
+      'width=600,height=800,scrollbars=yes'
+    )
 
     try {
       if (authConnector && socialProvider) {
         const { uri } = await authConnector.provider.getSocialRedirectUri({
```

Comment:
What if there is an error in this request? Would the popup be still open? If yes I think we should close it